andela/healthchecks_vulcan
# Ch hosting on heroku

#### What does this PR do? 
Sets common settings as environment variables. Settings code is DRY code and easily integrates with Heroku and the localhost.

#### Description of Task to be completed? 
Host the site on Heroku.
Have `SITE_ROOT`, `HOST`, `DATABASE_URL` as environment variables.
Have `DJMAIL_REAL_BACKEND` as a common setting
Change the name `local_settings.py` to a more descriptive name `dev_settings.py`

#### How should this be manually tested? 
Run the following in your terminal
`export SITE_ROOT=http://localhost:8000`
`export HOST=localhost`
`export DATABASE_URL=postgres://<user>:<password>@127.0.0.1:5432/<dbname>`
If you wish to override some of the settings in `settings.py`, create a file `dev_settings.py` in the same folder as `settings.py` i.e `hc`.
Run the application by running the following command in your terminal `python manage.py runserver`.
The app should run in your localhost, port 8000.

#### Any background context you want to provide? 
You might want to save this
`SITE_ROOT=http://localhost:8000`
`HOST=localhost`
`DATABASE_URL=postgres://<user>:<password>@127.0.0.1:5432/<dbname>`
in your `dev_settings.py` for your app to run well. Alternatively you could have them in your `.bash_profile` 

#### What are the relevant pivotal tracker stories? 
`#139617795`
